_maps/skyrat/automapper/automapper_config.toml

```diff
@@ -459,3 +459,102 @@ directory = "_maps/bubber/automapper/templates/generic/"
 required_map = "CentCom_skyrat_z2.dmm"
 coordinates = [72, 183, 2]
 trait_name = "CentCom"
+
+
+## Book of kindred display cases
+
+#DeltaStation2
+[templates.deltastation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "DeltaStation2.dmm"
+coordinates = [186, 105, 1]
+trait_name = "Station"
+
+#IceBoxStation
+[templates.iceboxstation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "IceBoxStation.dmm"
+coordinates = [175, 140, 3]
+trait_name = "Station"
+
+#LimaStation
+[templates.limastation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "LimaStation.dmm"
+coordinates = [99, 128, 2]
+trait_name = "Station"
+
+#KiloStation
+[templates.kilostation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "KiloStation.dmm"
+coordinates = [97, 128, 1]
+trait_name = "Station"
+
+#Birdshot
+[templates.birdshot_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "birdshot.dmm"
+coordinates = [80, 145, 1]
+trait_name = "Station"
+
+#Blueshift
+[templates.blueshift_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "Blueshift.dmm"
+coordinates = [95, 127, 1]
+trait_name = "Station"
+
+#NorthStar
+[templates.northstar_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "north_star.dmm"
+coordinates = [156, 138, 4]
+trait_name = "Station"
+
+#VoidRaptor
+[templates.voidraptor_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "VoidRaptor.dmm"
+coordinates = [146, 137, 1]
+trait_name = "Station"
+
+#Biodome
+[templates.biodome_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "biodome.dmm"
+coordinates = [116, 102, 1]
+trait_name = "Station"
+
+#TramStation
+[templates.tramstation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "tramstation.dmm"
+coordinates = [49, 103, 2]
+trait_name = "Station"
+
+#BurgerStation
+[templates.burgerstation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "burgerstation.dmm"
+coordinates = [75, 117, 1]
+trait_name = "Station"
```

Comment:
We own this map, and this map is deleted on the codebase. Moonstation is the updated version but it's not merged yet, so I'll ask Burger to update moonstation with it.

Reply:
Was just more neat to have all the related map changes in one thing, but I'll revert the changes for these maps

Reply:
Yeah I get it, but I want to limit the automapping on things we don't need it for. If we need some level of enforced consistency, then a mapping spawner can be coded in where you can change the contents without touching map files